File: app/src/main/java/com/example/fooddeuk/activity/Divider.java
package com.example.fooddeuk.activity;

import android.content.Context;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.view.View;

import com.example.fooddeuk.R;

/**
 * Created by heo on 2018. 2. 22..
 */

public class Divider extends View {
    public Divider(Context context) {
        super(context);
        inflate(getContext(), R.layout.custom_divider,null);
    }

    public Divider(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public Divider(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }


}

File: app/src/main/java/com/example/fooddeuk/custom/Divider.kt
package com.example.fooddeuk.custom

import android.content.Context
import android.support.v4.content.ContextCompat
import android.util.AttributeSet
import android.widget.LinearLayout
import com.example.fooddeuk.R

/**
 * Created by heo on 2018. 3. 17..
 */

class Divider @JvmOverloads constructor(
        context: Context,
        attrs: AttributeSet? = null,
        defStyleAttr: Int = 0
) : LinearLayout(context, attrs, defStyleAttr) {


    init {
        this.setBackgroundColor(ContextCompat.getColor(this.context, R.color.silver))
    }

}
